pandas/_libs/window.pyx

```diff
@@ -14,6 +14,7 @@ cnp.import_array()
 
 
 cdef extern from "../src/headers/cmath" namespace "std":
```

Comment:
@chris-b1 @jbrockmendel this changed between 0.23.x and master. On master we use

```cython
cdef extern from "src/headers/cmath" namespace "std":
```

But here we use `../src/headers/cmath`. Does this look correct for 0.23.x?

Reply:
This looks like something I would have nit-picked about.  The "../" is inaccurate in this context.  Evidently the cython/gcc compiler doesn't mind either way, but I'm pretty sure the one in master is More Correct.